cms/templates/home_page.html

```diff
@@ -10,16 +11,29 @@
 {% endblock %}
 
 {% block content %}
-    <div class="hero-container" style="background-image: url('{% wagtail_img_src page.hero%}')">
-      <div class="container hero-content">
-        <div class="hero-title">
-          {% if page.hero_title %}
-            <h1>{{ page.hero_title }}</h1>
-          {% endif %}
-          {% if page.hero_subtitle %}
-            <h2>{{ page.hero_subtitle }}</h2>
-          {% endif %}
-        </div>
-      </div>
-    </div>
-{% endblock %}
\ No newline at end of file
+<div class="course-section">
```

Comment:
You're completely removing a section of the page here (see `hero-container` in the removed part of the diff). This is a section that was added recently, it's outlined in the designs, and it needs to be there.

Reply:
Did not get this point.

Reply:
@asadiqbal08 This PR: https://github.com/mitodl/mitxonline/pull/35/files#diff-153077c90d135769287ff8c6025f219227d11fa426cc1da35b348a14050fe3d7

I know this PR was handed off to you and it might not have been you that rebased this on the main branch. Basically in the process of merging this with the code in master, all of the hero section code has been deleted from this template. Everything you see in that PR needs to remain in this template